docs/content/docs/data-structures/tree.md

```diff
@@ -547,7 +545,7 @@ on<K extends keyof TreeChangeEvents>(
 	): () => void;
 ```
 
-`Tree.on` assigns the specified `listener` function to the specified `eventType` for the specified `node`. The `node` can be any node of the tree. The `eventType` can be either "treeChanged" or "nodeChanged". `nodeChanged` fires whenever the given node changes. `treeChanged` fires whenever the given node or any of the nodes in its child subtree changes.
+`Tree.on` assigns the specified `listener` function to the specified `eventType` for the specified `node`. The `node` can be any node of the tree. The `eventType` can be either "treeChanged" or "nodeChanged". `nodeChanged` fires whenever the given node changes (values of the fields in the node). `treeChanged` fires whenever the given node or any of the nodes in its child subtree changes.
```

Comment:
Nit: I would recommend line-breaking Markdown contents along sentence boundaries. Makes future changes easier to review.
https://github.com/microsoft/FluidFramework/wiki/Markdown-Best-Practices#line-breaks-along-sentence-boundaries